src/scheduler.js

```diff
@@ -112,6 +113,18 @@ class Scheduler extends events.EventEmitter {
         taskId: hook.nextTaskId,
         time: new Date(),
       };
+
+      try {
+        await this.LastFire.create({
+          hookGroupId: hook.hookGroupId,
+          hookId: hook.hookId,
+          taskId: lastFire.taskId,
+          taskCreateTime: lastFire.time,
+        });
+      } catch (err) {
+        debug('Failed to append lastfire with err: %s', err);
```

Comment:
you can send this to `this.monitor.reportError(err)`, so it gets handled as an exception we can see.